src/on-premises/config/env/connect.md

```diff
@@ -0,0 +1,101 @@
+## Bimdata Connect
+
+### URLs of Bimdata apps
+
+| Variables    | Default value                           | Description                |
+|--------------|-----------------------------------------|----------------------------|
+| API_URL      | "http://localhost:8000"                 | Bimdata API URL.           |
+| SITE_URL     | "http://localhost:8001"                 | Bimdata Connect URL.       |
+| DOC_URL      | "https://developers-staging.bimdata.io" | Bimdata Documentation URL. |
+| PLATFORM_URL | "https://platform-staging.bimdata.io"   | Bimdata Platform URL.      |
```

Comment:
Default values shouldn't be bimdata URLs in on-premise (except for documentation)

Reply:
In this part of the documentation, it list all the environment variables for each container. This are the default values if the variable is not provided to the container. That's because of this kind of things that we spoke about building specific images for the on-premises some time ago.